src/test/java/tkachuk/sudoku/SudokuTest.java

```diff
@@ -0,0 +1,106 @@
+package tkachuk.sudoku;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SudokuTest
+{
+    @Test
+    public void getErrors()
+    {
+
+        // board with errors
+        int[][] nums = {
+                {4, 3, 5, 2, 6, 9, 7, 8, 1},
+                {6, 8, 2, 5, 7, 1, 4, 9, 3},
+                {1, 9, 7, 8, 3, 4, 5, 6, 2},
+                {8, 2, 6, 1, 9, 5, 3, 4, 7},
+                {3, 5, 4, 7, 8, 2, 9, 1, 8},
+                {9, 7, 1, 2, 4, 3, 6, 2, 5},
+                {5, 1, 9, 3, 2, 6, 8, 7, 4},
+                {2, 4, 8, 9, 5, 7, 1, 3, 6},
+                {7, 6, 3, 4, 1, 8, 2, 5, 9}};
+
+        Sudoku board = new Sudoku(nums);
+
+        board.getErrors();
```

Comment:
No you misunderstood me. 

I want two individual tests. One for a board with errors and one for a board without errors.


What I don't want you doing is calling getErrors() that didn't do anything like here on line 31